assets/js/directives/buy-steps.directive.js

```diff
@@ -4,7 +4,6 @@ angular.module('walletApp')
 function buySteps () {
   const directive = {
     restrict: 'E',
-    replace: true,
```

Comment:
I removed this so I could use ng-hide="isPendingBankTransfer()" in the coinify-modal template.